content/INTRODUCTION.md

```diff
@@ -0,0 +1,19 @@
+**Requirements**: Basic knowledge in PHP
+
+**Duration**: 
+
+## Object-Oriented Programming (OOP) in PHP
+
+Object-Oriented Programming (OOP) in PHP is a programming language principle that helps in building complex and reusable web applications. 
+
+Object-Oriented Programming (OOP) in PHP is designed for:
+
+* learners who have completed `An Introduction to Basic PHP Programming` and `Intermediate PHP Programming`.
```

Comment:
I think this we can have this. I did not start these statements in uppercase since before we have a discussion with the statement that ends with a colon `:` and Bal said that for the continuing statements after the `:` should start with a lowercase that is what I remember but I think this will not change the meaning of the whole context. I will edit the content. Thanks.